ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/State.java

```diff
@@ -12,7 +13,11 @@
 @Getter
 public enum State {
 
-    @JsonProperty("Draft")
+    @JsonProperty("DRAFT")
```

Comment:
Can we make the json property match the value in the CCD config: `Draft`

Reply:
I think this needs to be fixed in the CCD generator code, I shall take a look

Reply:
I managed to get this to work as "Draft" by renaming the the enum to Draft